### ISSUE TITLE
[a11y] The pop-up error message displayed on page doesn't have minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by plabanTest1** [civiform/civiform#9616]
**Describe the bug**
The contrast on the error message between foreground and background colors doesn't meet GAR guidelines for minimum contrast ratio thresholds

**To Reproduce**

1. Go to https://qa.civiform.dev/programs
2. Click on DevTools
3. Select Log in as: Civiform Admin
4. Click on "Intermediaries" from the top navbar.
5. Under the Create new trusted intermediary , provide a Name in the text box.
6. Click on "Create" button. 
7. An error message is displayed to the top centre of the page.

**Expected behavior**
The contrast between foreground and background colors should meet the WCAG 2 AA minimum contrast ratio thresholds

**Screenshots**

![Image](https://github.com/user-attachments/assets/4e52916a-adac-4dc9-a779-d9255b6078cc)

**Desktop (please complete the following information):**

- OS: [iOS]
- Browser [Chrome]
- Version [e.g. 132.0.6834.111]

**Additional context**

**Comment by shreyachatterjee00** [comment]
@myfriendgu are we updating these error messages as part of the USWDS migration?
